MapboxNavigation/RouteMapViewController.swift

```diff
@@ -34,6 +34,14 @@ class RouteMapViewController: UIViewController {
         }
         return parent.pendingCamera
     }
+    var defaultCamera: MGLMapCamera {
```

Comment:
This is more like `tiltedCamera` or `cameraFacingRoad`, since it’s a dynamic value rather than a fixed value.